tkg/client/init.go

```diff
@@ -273,6 +273,16 @@ func (c *TkgClient) InitRegion(options *InitRegionOptions) error { //nolint:funl
 		targetClusterNamespace = options.Namespace
 	}
 
+	if config.IsFeatureActivated(constants.FeatureFlagPackageBasedCC) {
+		// for cc based cluster, check if tkr/cbt is reconciled to avoid package not found error by addons-controller
+		tkrVersion, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableTkrName)
+		log.Infof("Checking Tkr %s in bootstrap cluster...", tkrVersion)
+		_, err = bootStrapClusterClient.GetTanzuKubernetesReleases(tkrVersion)
```

Comment:
I am not sure if the function GetTanzuKubernetesReleases does a retry. But we need to do some amount of retries since it might take some time for tkr to be available.